ethereum-network-partition-test/main_test.go

```diff
@@ -38,17 +38,16 @@ const (
 
 	nodeInfoPrefix = "NODES STATUS -- |"
 
-	ethModuleId    = "eth-module"
-	ethModuleImage = "kurtosistech/eth2-merge-kurtosis-module:0.7.2"
+	eth2StarlarkModule = "github.com/kurtosis-tech/eth2-module"
 
 	// must be something greater than 4 to have at least 2 nodes in each partition
 	numParticipants = 4
 
 	participantsPlaceholder = "{{participants_param}}"
 	//participantParam        = `{"elType":"geth","elImage":"ethereum/client-go:v1.10.25","clType":"lodestar","clImage":"chainsafe/lodestar:v1.1.0"}`
-	participantParam     = `{"elType":"geth","elImage":"ethereum/client-go:v1.10.25","clType":"lighthouse","clImage":"sigp/lighthouse:v3.1.2"}`
+	participantParam     = `{"el_client_type":"geth","el_client_image":"ethereum/client-go:v1.10.25","cl_client_type":"lighthouse","cl_client_image":"sigp/lighthouse:v3.1.2"}`
```

Comment:
Whoaaa hang on, are Pari/the Ethereum team/the client teams aware of this?? This is now a huge break in all their existing scripts, etc. wrapped around the module!!

Reply:
Haven't made them aware of this yet. I demo on Friday. The types were originally protobuf types and it made sense to have snake_case in there. Are you against the casing or against the change of names (elType -> el_type -> el_client_type)? What is your preference?

I don't have strong opinions here.

Reply:
the snake case is great if we were in a world with no users already; it's the breaking change that would require all users to snake-case all their input params. if the users are okay with it it's fine by me, but we'll want their assnet

Reply:
how do I talk to all the users? I am talking to Pari on Friday.

Reply:
Paritosh and Marius seem to be fine with it and say that switching shouldn't be a problem.

Reply:
@mieubrisse will still need your approval on this!